src/data/polynomial/field_division.lean

```diff
@@ -356,5 +357,52 @@ begin
   rw [← C_inj, this, C_0],
 end
 
+lemma prod_multiset_root_eq_finset_root {p : polynomial R} (hzero : p ≠ 0) :
+(multiset.map (λ (a : R), X - C a) p.roots).prod =
```

Comment:
Can you indent the statement? Github won't let me do a 2-line suggestion, so here's two separate ones:
```suggestion
  (multiset.map (λ (a : R), X - C a) p.roots).prod =
```

Reply:
I have indentented the statement. I will think about the other comments on Monday